### ISSUE TITLE
Please indicate status of Python 3 support on website

### ISSUE THREAD

**Issue by sidneycadot** [clade/PyDAQmx#14]
Hi,

A suggestion: it would be useful to have a brief statement concerning Python 3 support on the project's website.

The installation page currently says: "It has been reported that the package works on Python 3 using 2to3.". That suggests it _may_ work, but making it work (and fixing problems) is not a priority. Is this correct? If so, it will be useful for potential users to be aware of this.

**Comment by fbender** [comment]
Yeah, updated classifiers with supported Python version whould be great, too!

I also get the impression that 2to3 is actually no longer needed (the website should be updated to reflect this). Though I cannot test that right now.

**Comment by clade** [comment]
The package works on Python 3. 2to3 is now run by the setup script automatically. So there is no need to explicitly call 2to3, an a simple "python setup.py install" works. But it uses 2to3.
